commands/roll.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (bb8) {
+    console.log('Let\'s Roll!!');
+
+    setInterval(function () {
+        var direction = Math.floor(Math.random() * 360);
+        bb8.roll(150, direction);
```

Comment:
Ha, that's the thing I was talking about with @xepps yesterday! I remember that from primary school :smile: